internal/pkg/util/bin/bin.go

```diff
@@ -68,9 +82,15 @@ func findOnPath(name string) (path string, err error) {
 func findFromConfigOrPath(name string) (path string, err error) {
 	cfg := apptainerconf.GetCurrentConfig()
 	if cfg == nil {
-		cfg, err = apptainerconf.Parse(buildcfg.APPTAINER_CONF_FILE)
-		if err != nil {
-			return "", errors.Wrap(err, "unable to parse apptainer configuration file")
+		if strings.HasSuffix(os.Args[0], ".test") {
```

Comment:
Instead of having this directly in the code, I would put the parsing and initialization of current config directly in `_test.go` files